molecule/driver/docker.py

```diff
@@ -111,11 +111,21 @@ class Docker(Driver):
             buildargs:
                 http_proxy: http://proxy.example.com:8080/
 
-    If specifying the `CMD`_ directive in your ``Dockerfile.j2`` or consuming a
-    built image which declares a ``CMD`` directive, then you must set
-    ``override_command: False``. Otherwise, Molecule takes care to honour the
-    value of the ``command`` key or uses the default of ``bash -c "while true;
-    do sleep 10000; done"`` to run the container until it is provisioned.
+    Molecule takes care to honour the value of the ``entrypoint`` and
+    ``command`` keys, or defalt with ``''`` and ``bash -c "sleep infinity"``
+    to run the container until it is provisioned.
+
+    When attempting to utilize base image original ``ENTRYPOINT`` and ``CMD``
+    behavior, make sure to unset the ``entrypoint`` and ``command``. An
```

Comment:
Since the `entrypoint` control is a new feature, why not default it to `~` so users can choose to override it if desired, otherwise, leave it alone?

Reply:
Depends on how we define the default behavior: molecule style sleep infinity for running test, or out of control depends on base image default?

Here I vote for molecule default with sleep infinity, because this should be most common molecule use cases: run and test our Ansible logic, but not testing the base image original behavior.

Therefore the ‘’ (ie /bin/sh -c) and /bin/bash -c “sleep infinity” is the only combo that could make any base image running with our expect default behavior in a manageable style.